SConstruct

```diff
@@ -58,6 +59,7 @@ AddOption(
     '--qt',
     dest='qt',
     type='string',
+    default='/usr',
```

Comment:
agreed to split in 2 commits.
For 4 spaces, it's not necessary but in this file 4 and 8 spaces are used.